gui/scripts/syncthing/core/controllers/syncthingController.js

```diff
@@ -995,6 +995,9 @@ angular.module('syncthing.core')
             $scope.currentFolder.devices.forEach(function (n) {
                 $scope.currentFolder.selectedDevices[n.deviceID] = true;
             });
+            if (typeof $scope.currentFolder.order === 'undefined') {
+              $scope.currentFolder.order = "random";
+            }
```

Comment:
I'm not super keen on this one - it makes it look like an undefined order would happen now and then, which it shouldn't for any existing folders, and we don't sanity check other folder attributes like this. I say skip it for the moment, just for clarity.